src/requirements/__test__/requirement-graph-builder.test.ts

```diff
@@ -22,14 +22,17 @@ const getAllCoursesThatCanPotentiallySatisfyRequirement = (
 };
 
 it('buildRequirementFulfillmentGraph phase 1 test 1', () => {
-  const { requirementFulfillmentGraph: graph } = buildRequirementFulfillmentGraph({
-    requirements,
-    userCourses: [CS3410, CS3420, MATH4710],
-    userChoiceOnFulfillmentStrategy: {},
-    userChoiceOnDoubleCountingElimination: [],
-    getAllCoursesThatCanPotentiallySatisfyRequirement,
-    allowDoubleCounting: () => false,
-  });
+  const graph = buildRequirementFulfillmentGraph(
+    {
+      requirements,
+      userCourses: [CS3410, CS3420, MATH4710],
+      userChoiceOnFulfillmentStrategy: {},
+      userChoiceOnDoubleCountingElimination: [],
+      getAllCoursesThatCanPotentiallySatisfyRequirement,
+      allowDoubleCounting: () => false,
+    },
+    /* keepCoursesWithoutDoubleCountingEliminationChoice */ true
+  );
```

Comment:
For my own understanding: we have the testing flag `keepCoursesWithoutDoubleCountingEliminationChoice` as `true` because it we didn't have this flag at all, we would accidentally not match the possible courses to the requirement in phase 1? For example, the edges between the courses `CS3410` and `CS3420` and `'CS3410/CS3420'` would not exist without this testing flag because the user choice for these courses would be null?